src/ImageSharp/Formats/Png/PngScanlineProcessor.cs

```diff
@@ -387,12 +387,12 @@ internal static class PngScanlineProcessor
             ref Rgb24 rgb24SpanRef = ref MemoryMarshal.GetReference(rgb24Span);
             for (int x = 0; x < header.Width; x++)
```

Comment:
Would it not be better in these insytances to use `nint` in the outer loop since we're not touching indexers?

```c#
for (nint x = 0; x < (uint)header.Width; x++)
```